packages/react-core/src/components/Card/Card.tsx

```diff
@@ -12,20 +12,18 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   className?: string;
   /** Sets the base component to render. defaults to article */
   component?: keyof JSX.IntrinsicElements;
-  /** Modifies the card to include hover styles on :hover */
+  /** @deprecated Modifies the card to include hover styles on :hover */
   isHoverable?: boolean;
-  /** @beta Modifies the card to include hoverable-raised styles on :hover, this styling is included by default with isSelectableRaised */
-  isHoverableRaised?: boolean;
   /** Modifies the card to include compact styling. Should not be used with isLarge. */
   isCompact?: boolean;
   /** Modifies the card to include selectable styling */
   isSelectable?: boolean;
   /** Modifies the card to include selected styling */
   isSelected?: boolean;
-  /** @beta Modifies the card to include selectable-raised styling and hoverable-raised styling */
-  isSelectableRaised?: boolean;
-  /** @beta Modifies the card to include selected-raised styling */
-  isSelectedRaised?: boolean;
+  /** @beta Specifies the selectable styling variant */
+  selectableVariant?: 'legacy' | 'raised';
```

Comment:
legacy implies the non raised variant will eventually be deprecated.  is that the case @mcarrano?

Reply:
I figured it means it's not preferred and the 'old but supported' way

Reply:
Ok I see my confusion. I thought the selectable variant set the card to selectable.  That being said.  I think the API is a little confusing.  May we can rename the prop tp `selectableVariantStyling`.
I would also suggest not calling the option  `legacy`.  It is not really descriptive.

Reply:
So I was thinking about this more.  Would it make sense to deprecate the `isSelectable` prop and let the selectableVariant prop not only set the styling, but set it selectable too?  I think that would make the API more straight forward.  Similar to what @mcoker  was suggesting in his comment about combining props.

Reply:
How about `selectStyle` or `selectableStyle` over using `variant`, since the function of the select remains the same?